### ISSUE TITLE
webview2 screen sharing window size[Feature]: 

### ISSUE THREAD

**Issue by lsyfg** [MicrosoftEdge/WebView2Feedback#4192]
### Describe the feature/enhancement you need

I would like to provide screen sharing window size and coordinates, but when using applications such as WPF window loading, it is not possible to fill the entire window and there is extra space

### The scenario/use case where you would use this feature

Using WPF or Winform applications to load forms that cannot be filled

### How important is this request to you?

Impactful. My app's user experience would be significantly compromised without it.

**Comment by victorhuangwq** [comment]
> I would like to provide screen sharing window size and coordinates, but when using applications such as WPF window loading, it is not possible to fill the entire window and there is extra space

Could you rephrase / clarify what you are asking for here? It would be good if you have some diagrams if that helps

**Comment by lsyfg** [comment]
WPF application form loading webview2 screen sharing window, unable to fill WPF form due to unknown window size and coordinates
![image](https://github.com/MicrosoftEdge/WebView2Feedback/assets/6143343/9a0d5e09-44da-42f7-9806-e7b3b80b1b51)

**Comment by lsyfg** [comment]
Can screen sharing windows obtain form size and coordinates？

**Comment by lsyfg** [comment]
The WPF application form loads a screen sharing window. Due to the inability to obtain the size and coordinates of the screen sharing window, the form cannot be filled in the application form, leaving extra space. How to obtain the size and coordinates of the screen sharing window?
![QQ截图20231201091325](https://github.com/MicrosoftEdge/WebView2Feedback/assets/6143343/6f54b871-5bc5-4d66-9114-d145de77a498)

**Comment by victorhuangwq** [comment]
@srwei do you know if it's possible to obtain the size of the screen share popup right now?

**Comment by lsyfg** [comment]
@srweiMy application form size is fixed and not full screen. It only loads screen sharing windows, I need to know its size to fill my form

**Comment by lsyfg** [comment]
@srwei My application form size is fixed and not full screen. It only loads screen sharing windows, I need to know its size to fill my form

**Comment by lsyfg** [comment]
@victorhuangwq Hi, can we currently achieve the window size for obtaining screen sharing。I really need this feature because it directly affects aesthetics and effectiveness

**Comment by lsyfg** [comment]
@victorhuangwq Is there a plan to implement this issue in the new version

**Comment by lsyfg** [comment]
@victorhuangwq Is there a plan to implement this issue in the new version

**Comment by victorhuangwq** [comment]
Tracking it as it is not possible at the moment

**Comment by srwei** [comment]
@lsyfg  Sorry for the late response. There is currently no way to get the window size for the screen capture UI. We will look into adding this as part of the feature to have a custom UI for Screen Capture in the future.

**Comment by lsyfg** [comment]
I'm looking forward to it. You've worked hard and I wish you all the best in your work
